substrate-archive/src/wasm_tracing.rs

```diff
@@ -164,6 +164,7 @@ impl TraceHandler {
 
 	// we need this because we don't know the values until after tracing has been executed
 	/// Returns true if a span is part of an enabled Target. Checks WASM in addition to the spans target.
+	#[allow(clippy::suspicious_operation_groupings)]
```

Comment:
Is this about silencing the complaint on line 175 (suggestion: "use t.level")? If so that seems to be a clippy bug and not something worth silencing like this imo.

Reply:
Yes, it seems to be a clippy bug (rust-lang/rust-clippy#6722) of rust 1.50, which needs to be fixed by the upstream

Reply:
@dvdplm PTAL